nifi-test/pom.xml

```diff
@@ -0,0 +1,145 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>nifi</artifactId>
+        <groupId>org.apache.nifi</groupId>
+        <version>1.8.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.thinkbiganalytics</groupId>
+    <artifactId>nifi-test</artifactId>
```

Comment:
nifi-test seems too generic for this.  why not nifi-test-harness ?

Reply:
+1

Reply:
It makes sense. I'll fix in my fork and create a new pull request later on.